src/server/language.py

```diff
@@ -34,7 +34,7 @@ class Language(object):
     JA = _Language('日本語', 'ja')
     PT = _Language('Português', 'pt')
     ZH_CN = _Language('简体中文', 'zh-CN')
-
+    ZH_CHT=_Language('繁體中文', 'zh-CHT')
```

Comment:
Missed a space here:

```suggestion
    ZH_CHT= _Language('繁體中文', 'zh-CHT')
```

Reply:
revised it.